reports/Michigan.md

```diff
@@ -163,6 +163,20 @@ id: mi-grandrapids-2
 * https://twitter.com/sparrowsongs_/status/1266948928966139904
 
 
+### Police assault peaceful protesters, among them Breonna Taylor's family members | July 12th
+
+City of Grand Rapids Police Department officers seen pre-emptively pushing, pulling, and shoving peaceful protesters, as they circle around an officer arresting or detaining a man.
+
+The detaining officer appears to force the man to his knees before the other officers finish encircling and view of the man is lost.
+
+Two different women, each in different videos, are shown falling or fallen over as a result of the shoving and pulling.
+
```

Comment:
You're missing a couple things here: tags and ID. 

You'll want to look in the other Grand Rapids incidents and find the highest incremented ID and ++ it. I know it's not a perfect system, but we have something that works with non-forked branches, it just hasn't been finagled to work with forked repos.

And tags: check out some other incidents for reference. Every incident should have at least one "victim" tag (in this case it would be protestor). This should also mention "arrest", "shove", "push". Check [here](https://github.com/2020PB/police-brutality/blob/master/docs/possible_tags.md) for all the currently supported tags